scripts/travis/cleanup.js

```diff
@@ -84,20 +56,51 @@ const getCurrentRevs = (db, response) => {
   return db.allDocs({ keys: ids });
 };
 
+const queryReleases = (db, daysToKeep) => {
+  return db
+    .query('builds/releases', {
+      startkey: [ 'branch', 'medic', 'medic' ],
+      endkey: [ 'branch', 'medic', 'medic', getEndDate(daysToKeep) ],
+      limit: MAX_BUILDS_TO_DELETE
+    })
+    .then(response => getCurrentRevs(db, response));
+};
+
+const getTestingBuilds = db => {
+  console.log('Querying for old testing builds...');
+  return queryReleases(db, DAYS_TO_KEEP_TEST);
+};
+
+const getBranchBuilds = db => {
+  console.log('Querying for old branch builds...');
+  return queryReleases(db, DAYS_TO_KEEP_BRANCH);
+};
+
+const getBetaBuilds = db => {
+  console.log('Querying for old beta releases...');
+  return db
+    .query('builds/releases', {
+      startkey: [ 'beta', 'medic', 'medic', 10000 ],
+      endkey: [ 'beta', 'medic', 'medic', 0 ],
```

Comment:
Do we need the 0 here?

Reply:
It makes me feel more comfortable. Would you prefer it without?

Reply:
:) no, it's fine :)